docs/drivers.md

```diff
@@ -21,6 +21,10 @@ the host PATH:
 The KVM2 driver is intended to replace KVM driver.
 The KVM2 driver is maintained by the minikube team, and is built, tested and released with minikube.
 
+NOTE: Currently the following instruction doesn't work for
+Ubuntu prior to 18.04 because the docker-machine-driver-kvm2 binary
+provided by the URL needs libvirt 3.0.0 or later.
+
```

Comment:
This should specify a workaround, namely that you can build the docker-machine-driver-kvm2 binary yourself.